openmetadata-ui/src/main/resources/ui/src/jsons/connectionSchemas/connections/database/mysqlConnection.json

```diff
@@ -9,8 +9,8 @@
     "mySQLType": {
       "description": "Service type.",
       "type": "string",
-      "enum": ["MySQL"],
-      "default": "MySQL"
+      "enum": ["Mysql"],
```

Comment:
Hi @darth-coder00 sorry if I am late to the party. Why do we have the JSON schemas in the UI?

Reply:
got it, thanks